src/__tests__/JWT-test.ts

```diff
@@ -1,7 +1,7 @@
 import { createJWT, verifyJWT, decodeJWT, createJWS, verifyJWS, resolveAuthenticator, NBF_SKEW } from '../JWT'
 import { TokenVerifier } from 'jsontokens'
-import SimpleSigner from '../SimpleSigner'
-import NaclSigner from '../NaclSigner'
+import SimpleSigner from '../signers/SimpleSigner'
```

Comment:
Is moving the import for these deprecated Signer a breaking change or does Veramo export them globally once more? Otherwise I feel we might even drop the deprecated Implementations. :)

Reply:
It is not a breaking change since they are re-exported from `index.ts`